src/apps/reservation-list/reservation-material/reservation-info.tsx

```diff
@@ -81,10 +81,11 @@ const ReservationInfo: FC<ReservationInfoProps> = ({
         color={success as string}
         percent={100}
         info={getInfo()}
-        label={[
-          pickupLibrary,
-          pickupNumber || t("reservationListDigitalPickupText")
-        ]}
+        label={
+          !isDigital
```

Comment:
I know it is a small thing, but I would prefer if we removed the `not` check and just switched the two arrays around.